src/collective/behavior/banner/banner.py

```diff
@@ -53,6 +57,28 @@ class IBanner(model.Schema):
         required=False,
     )
 
+    banner_url = schema.URI(
+        title=_(u'label_banner_url', default=u'Video URL'),
+        description=u'''
+        If you want the banner for this item to show a video, enter an
+        external URL here.  YouTube and Vimeo are supported.  Note:
+        You can either supply an image, or a video URL, not both.
+        ''',
+        required=False,
+    )
+
+    # @invariant
+    # def validate_visual_content(data):
+    #     """
+    #     It is not allowed to add an image and an external url at the same time.
+    #     """
+    #     if data.banner_image and data.banner_url:
+    #         raise Invalid(_(
+    #             u'error_message_links',
+    #             default=u'It\'s not possible to add an image and an '
+    #                     u'external url at the same time')
+    #         )
+
```

Comment:
I want to add it back later once I figure out how to make the error
highlight the appropriate field.

On Fri, Nov 27, 2015 at 1:45 PM Gil Forcada Codinachs <
notifications@github.com> wrote:

> In src/collective/behavior/banner/banner.py
> <https://github.com/collective/collective.behavior.banner/pull/4#discussion_r46071038>
> :
>
> > +        ''',
> > +        required=False,
> > +    )
> > +
> > +    # @invariant
> > +    # def validate_visual_content(data):
> > +    #     """
> > +    #     It is not allowed to add an image and an external url at the same time.
> > +    #     """
> > +    #     if data.banner_image and data.banner_url:
> > +    #         raise Invalid(_(
> > +    #             u'error_message_links',
> > +    #             default=u'It\'s not possible to add an image and an '
> > +    #                     u'external url at the same time')
> > +    #         )
> > +
>
> any special reason to not remove the code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/collective/collective.behavior.banner/pull/4/files#r46071038>
> .
>